runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/Users/Users.thunks.ts

```diff
@@ -48,16 +48,16 @@ export const update = createAsyncThunk<UserDto, PartialUserDto>(
 
 export const activate = createAsyncThunk<void, ActivateUserDto>(
     'users/activate',
-    (activateData) => axios.post<ActivateUserDto>(`/api/scheduler/users/${activateData.id}/activate`, activateData)
+    (activateData) => axios.patch<ActivateUserDto>(`/api/scheduler/users/${activateData.id}/activation`, activateData)
```

Comment:
should be a verb, like a name of function, so 'activation' is incorrect, because it's a noun